src/PolyhedralGeometry/Cone/constructors.jl

```diff
@@ -91,6 +91,13 @@ function ==(C0::Cone{T}, C1::Cone{T}) where T<:scalar_types
     return Polymake.polytope.equal_polyhedra(pm_object(C0), pm_object(C1))
 end
 
+# we do not want to force a convex hull computation and even if we did, that
+# would not give a normal form
```

Comment:
This explanation seems to be missing something... it seems to presuppose that the reader would expect the hash to compute a convex hull? How about something like this:
```suggestion
# For a proper hash function for cones we should use a "normal form",
# which would require a potentially expensive convex hull computation
# (and even that is not enough). But hash methods should be fast, so we
# just consider the ambient dimension and the precise type of the cone.
```